torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1241,21 +1241,12 @@ at::Tensor XLANativeFunctions::empty_strided_symint(
   bool is_size_dynamic = !size.has_value();
   c10::optional<at::IntArrayRef> stride = c10::asIntArrayRefSlowOpt(sym_stride);
   bool is_stride_dynamic = !stride.has_value();
-  at::Tensor t =
-      empty_symint(sym_size, dtype, layout, device, pin_memory, c10::nullopt);
-  if (is_size_dynamic || is_stride_dynamic) {
-    // As XLATensor doesn't have a storage, it should not care about the memory
-    // format or how to jump to the next element (strides). So the term stride
-    // does not mean much to us. The size of the tensor has been set by the
-    // above `empty_symint` so we feel it is ok to return here. The reason why
-    // we only do it for dynamic size/stride is to limit the change and preserve
-    // the old behavior.
-    return t;
-  } else {
-    return torch_xla::XLANativeFunctions::as_strided_copy(t, size.value(),
-                                                          stride.value(),
-                                                          /*storage_offset=*/0);
-  }
+  // As XLATensor doesn't have a storage, it should not care about the memory
+  // format or how to jump to the next element (strides). So the term stride
+  // does not mean much to us. The size of the tensor has been set by the
+  // above `empty_symint` so we feel it is ok to return here.
+  return empty_symint(sym_size, dtype, layout, device, pin_memory,
```

Comment:
@alanwaketan to confirm

Reply:
@alanwaketan has already approved the unconditional case in https://github.com/pytorch/xla/pull/5239..